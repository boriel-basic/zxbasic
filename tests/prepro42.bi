#define X 0

#if X
print X
#endif

