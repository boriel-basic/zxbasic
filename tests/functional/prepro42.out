#line 1 "prepro42.bi"


#line 6 "prepro42.bi"

